library/src/interactor_impl.cxx

```diff
@@ -1049,7 +1049,6 @@ bool interactor_impl::playInteraction(const std::string& file)
 
     std::string cleanFile = vtksys::SystemTools::CollapseFullPath(file);
     this->Internals->Recorder->SetFileName(cleanFile.c_str());
-    this->Internals->Window.UpdateDynamicOptions();
```

Comment:
TBH I dont know why this was ever needed at all, it dates back to my original implementation of the options.

In any case, options are dynamically updated on render, and with this change I fall into a segfault in VTK.
I fixed the segfault in VTK https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11740, but since this code was not needed at all, I just removed it instead of adding version checks

Reply:
Ok I don't remember the details but I think this was also introducing some duplicated warnings in some cases.
Anyway, good to see useless code removed.